packages/matchbox/src/components/Banner/styles.js

```diff
@@ -0,0 +1,94 @@
+import { tokens } from '@sparkpost/design-tokens';
+import { ErrorOutline, CheckCircleOutline, InfoOutline } from '@sparkpost/matchbox-icons';
+
+export function container(props) {
+  switch (props.status) {
+    case 'success':
+      return `
+        background: ${tokens.color_green_100};
+        border: 1px solid ${tokens.color_green_600};
+      `;
+    case 'warning':
+      return `
+        background: ${tokens.color_yellow_100};
+        border: 1px solid ${tokens.color_yellow_500};
+      `;
+    case 'danger':
+      return `
+        background: ${tokens.color_red_100};
+        border: 1px solid ${tokens.color_red_700};
+      `;
+    // Gray banner no longer exists as of 4.0.0
+    case 'default':
+    default:
+      return `
+        background: ${tokens.color_blue_100};
+        border: 1px solid ${tokens.color_blue_600};
+      `;
+  }
+}
+
+export const statusIcons = {
```

Comment:
Looks like some funky things start to happen on smaller viewports:
![Screen Shot 2020-02-20 at 9 20 27 AM](https://user-images.githubusercontent.com/3613392/74943329-5a2ed180-53c2-11ea-8f65-4783abdf9445.png)

Reply:
This ones a tough example because the button text is causing the overflow, I'll update the story though